package.json

```diff
@@ -111,7 +111,7 @@
     "glob": "^7.1.1",
     "jest": "^26.6.3",
     "ky": "^0.12.0",
-    "lerna": "^3.16.4",
+    "lerna": "^6.4.1",
     "link-parent-bin": "^1.0.0",
```

Comment:
link-parent-bin was removed on terra-toolkit by @saket2403  : https://github.com/cerner/terra-toolkit/pull/769/files
not able to find why this was removed since PR description was not updated with reason for removing `link-parent-bin`. 
Can you check with @saket2403 and update here as well

Reply:
Removed as not required https://github.com/cerner/terra-framework/pull/1595/commits/0794eb6efdf8cd80184b9949d99cc3b28706969c